cmd/thermal-recorder/frameloop.go

```diff
@@ -33,9 +33,8 @@ type FrameLoop struct {
 	currentIndex  int
 	frames        []*lepton3.Frame
 	orderedFrames []*lepton3.Frame
-	zeroFrame     lepton3.Frame
 	bufferFull    bool
-	mu            sync.Mutex
+	oldest        int
```

Comment:
The mutex is gone but I don't see anything that provides goroutine safety now... what's up with that?